clients/java/src/main/java/io/camunda/zeebe/client/api/search/response/DecisionInstance.java

```diff
@@ -26,6 +26,11 @@ public interface DecisionInstance {
    */
   long getDecisionInstanceKey();
 
+  /**
+   * @return the id of the decision instance
```

Comment:
```suggestion
   * @return the ID of the decision instance
```

Nitpick to uppercase ID 👍